### PR TITLE
Fix #54 Refactor folder structure and rename some files.

### LINKED ISSUE #54: Refactor folder structure and rename some files.
Currently, we have a basic folder structure and we are adding every component to the component folder. We need to separate common and layout components into separate folders.
Also, we need to rename some files and change the file extension according to what value the file contains.

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Closes #54

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
@ashutosh1919, Please review the changes.

File: src/lib/shapes/data/polygonclippath.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const clipPathMap = {
```
Comment (ashutosh1919) [inline src/lib/shapes/data/polygonclippath.ts:1]: Can you please rename this file to `polygonClipPath.ts`?

**Review by ashutosh1919 — CHANGES_REQUESTED** [review]
Everything looks good. Just a small change.

File: src/lib/shapes/data/polygonclippath.ts (hunk 1 above)
Comment (Sachin-chaurasiya) [reply src/lib/shapes/data/polygonclippath.ts:1]: Sure.

**Pull request merged**